app/server/tests/db/sessionStore.test.ts

```diff
@@ -12,14 +12,16 @@ describe("Sessionstore", () => {
         exec: jest.fn()
     } as any;
     mockPipeline.hset = jest.fn().mockReturnValue(mockPipeline);
+    mockPipeline.hsetnx = jest.fn().mockReturnValue(mockPipeline);
```

Comment:
I think you shouldn't need this, as the method doesn't use pipeline

Reply:
right, that was left over from a previous attempt :upside_down_face: